Index: app/date_func.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/date_func.py	(date 1538757714000)
+++ app/date_func.py	(date 1538758708000)
@@ -7,10 +7,16 @@
 =======
 from random import choice
 
+<<<<<<< HEAD:app/date_func.py
 import holidays
 from holidays import holidays
 from quotes import *
 >>>>>>> date_func:date_func.py
+=======
+from app import holidays
+from app.holidays import holidays
+from app.quotes import *
+>>>>>>> date_func:app/date_func.py
 
 
 def countdown(holiday):
